cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowSpecification.java

```diff
@@ -55,381 +34,81 @@
  *  <pre>
  *    <code>
  *      {@literal @}Override
- *      public WorkflowSpecification configure() {
- *        return WorkflowSpecification.Builder.with()
- *        .setName("PurchaseHistoryWorkflow")
- *        .setDescription("PurchaseHistoryWorkflow description")
- *        .onlyWith(new PurchaseHistoryBuilder())
- *        .addSchedule(new Schedule("DailySchedule", "Run every day at 4:00 A.M.", "0 4 * * *",
- *                     Schedule.Action.START))
- *        .build();
+ *      public void configure() {
+ *        setName("PurchaseHistoryWorkflow");
+ *        setDescription("PurchaseHistoryWorkflow description");
+ *        addMapReduce("PurchaseHistoryBuilder");
+ *        addSchedule("DailySchedule");
```

Comment:
This needs to be changed right ? 
The program schedules will be in the application right ?